doc/CI_Hardware.md

```diff
@@ -0,0 +1,221 @@
+# Tock Hardware-in-the-Loop CI
+In addition to software tests, Tock allows platform maintainers to deploy CI runners for hardware platforms, to ensure continuous on-metal performance correctness. The hardware CI is federated, and maintainers for each hardware platform are responsible for ensuring that their hardware CI system is set up, accessible, and runs correctly.
+
+The Tock core team maintains the following hardware CI instances:
+- Nrf52840dk	UC San Diego; contact: Pat Pannuto <ppannuto@ucsd.edu>, Anthony Quiroga <aquiroga@ucsd.edu> 
+- Hail (WIP)	UC San Diego; contact: Pat Pannuto <ppannuto@ucsd.edu>, Anthony Quiroga <aquiroga@usd.edu>
+
+<!-- npm i -g markdown-toc; markdown-toc -i CI_Hardware.md -->
+<!-- toc -->
+<!-- tocstop -->
+
+![Workflow](tock/doc/images/workflow.png)
+
+# How To Set Up a New Instance
+
```

Comment:
Can we add a leader paragraph here, something to make clear that all of these directions are about setting up the local runner machine. Maybe something like:

> You will need a local machine to run the unit tests that is attached to the Tock development board under test.
>
> This section explains the software setup needed for this box. These directions assume you are using a Raspberry Pi, which will likely be easiest as there are already pins and drivers for GPIO, SPI, and I2C testing available for RPis, but in principle, any Linux-like platform should work.
> 
> **Note:** This machine should be on a stable, reliable network (i.e. a campus or office setup) to ensure it is always available to run tests. Also of note, runners may run arbitrary code from community-generated PRs, which may create a security risk. Once initial configuration is complete, the _only_ network access the runner should need is to Github servers, so this box can be firewalled accordingly.